app/components/Menu/index.js

```diff
@@ -60,28 +59,44 @@ export default class Menu extends React.PureComponent {
     const path = this.props.pathname.split('/');
     const getLocation = path[1] || path[0];
 
-    if (this.props.pathname.length > 1 && this.props.pathname !== '/multumim') {
-      document.getElementById(getLocation).parentNode.querySelector('.selected').classList.remove('selected');
-      document.getElementById(getLocation).firstElementChild.classList.add('selected');
-    } else {
-      // document.querySelector('#menu-list .selected').classList.remove('selected');
-    }
+    // if (this.props.pathname.length > 1 && this.props.pathname !== '/multumim') {
```

Comment:
Get class selected without dom manipulation :1st_place_medal:

Reply:
On it :P